webpay/pay/forms.py

```diff
@@ -92,7 +92,7 @@ def clean_req(self):
             raise forms.ValidationError(msg.BAD_ICON_KEY)
 
         for fn in msg.SHORT_FIELDS:
-            if (len(payload['request'].get(fn, '')) >
+            if (len(payload['request'].get(fn, None) or '') >
```

Comment:
hopefully the test will keep it from getting refectored out ;) but yeah I'll add a note. This is a typical string problem in Python - original code was sloppy.